client/core/core_test.go

```diff
@@ -4287,6 +4299,56 @@ func TestHandleTradeSuspensionMsg(t *testing.T) {
 	}
 }
 
+func setupOrderNotificationVerification(tCore *Core, t *testing.T) func() *OrderNote {
```

Comment:
I'd suggest a couple of changes here. 1) The name doesn't seem descriptive of the function, 2) How about a goroutine and a channel instead. Otherwise, your feed is just filling with notes while you're not running the `getOrderNote` function.

Something like ....

```
func orderNoteFeed(tCore *Core, t *testing.T) chan *OrderNote {
  ch := tCore.NotificationFeed()
  outChan := make(chan *OrderNote)
  go func(){
      for {
          select {
          case n := <-ch:
              ordNote, ok := n.(*OrderNote)
              if ok {
                  outChan <- ordNote
              }
          case <-tCtx.Done():
              return
          }
      }
  }()
  return outChan
}
```